RiotSharp/MatchEndpoint/Player.cs

```diff
@@ -10,6 +10,17 @@ public class Player
         internal Player() { }
 
         /// <summary>
+        /// Current platform ID.
+        /// </summary>
+        [JsonProperty("currentPlatformId")]
+        public string CurrentPlatformId { get; set; }
```

Comment:
can't we leverage platform here?

Reply:
I'm not sure ton understand the meaning.

Reply:
I mean replace the string by https://github.com/BenFradet/RiotSharp/blob/develop/RiotSharp/Misc/Platform.cs